R/setOrRetrieve.R

```diff
@@ -0,0 +1,108 @@
+.internal <- list2env(list(
+  recomputedHashtab = hashtab(type = "address", NULL), # unclear if "address" is the correct choice here
```

Comment:
This is indeed unclear to me too: What's your reason to choose `address` here? The docs say that `identical` is almost always the better choice, but here we are in a highly unusual setting, so I am willing to bet on your reasoning...

Reply:
my reasoning is that we're using R6 objects as hash, which are environments and are always unique (copies need to be made explicitly, but we basically don't do that). Perhaps this is a premature optimization though.